pulp_smash/tests/rpm/api_v2/test_repo_create.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+"""Test the API endpoints for RPM `repositories`_.
+
+The assumptions explored in this module have the following dependencies::
+
+    # test_create_sync.py
+    It is possible to create an RPM repo.
+    ├── It is impossible to create a duplicate relative url.
+    ├── It is possible to create an RPM repo with feed url. ← TEST THIS
+        ├── It is possible to sync repository from feed. ← TEST THIS
+    ├── It is possible to create an RPM repo without feed url.
+        ├── It is possible to upload single RPM file.
+        ├── It is possible to upload directory of RPM files.
+        ├── It is possible to upload ISO of RPM files.
+
+    # test_distribute.py
+    It is posible to publish an RPM repo with the "???" distributor.
+    ...
+
+.. _repositories:
+   http://pulp.readthedocs.org/en/latest/dev-guide/integration/rest-api/repo/cud.html
+
+"""
+from __future__ import unicode_literals
+
+import requests
+from pulp_smash.config import get_config
+from pulp_smash.constants import REPOSITORY_PATH
+from random import randint
+from unittest2 import TestCase
+
+
+_RPM_REPO_URL = 'http://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/zoo/'
+
+
+def _rand_str():
+    """Return a randomized string."""
+    return type('')(randint(-999999, 999999))
+
+
+class CreateFeedTestCase(TestCase):
```

Comment:
Would we be adding similar tests for different importers? In other words, should he name of this module OR the name of the class state this this test is valid for a yum importer only? This way, one could then add another module/class for puppet, docker, etc.